projects/js-packages/components/components/pricing-table/styles.module.scss

```diff
@@ -101,3 +109,42 @@
 .icon-cross {
 	--fill: var( --jp-red-50 );
 }
+
+.popover {
+	margin-left: auto;
+
+	.is-viewport-large & {
+		top: 1px;
+		margin: 0 var( --spacing-base );
+	}
+}
+
+.popover-icon {
+	fill: var( --fill, var( --jp-gray ) );
+	flex-shrink: 0;
+}
+
+.tos {
+	padding-left: var( --padding );
+	padding-right: var( --padding );
```

Comment:
Does this padding make sense on mobile?

Reply:
Actually, it might not need it, I removed it!